docs/metadata.md

```diff
@@ -22,378 +22,206 @@ searching for particular kinds of data.
-periodically call it to assertain liveness and optionally gather metrics as described in
-[Heartbeat and monitoring endpoint](ops_guidelines/data_provider_ops_guidelines.md#heartbeat-and-monitoring-endpoint)
+## Dataset metadata fields
 
-A hearbeat URL can be specified as a single key `heartbeat_url` with the value being the fully qualified URL at which 
-the hearbeat response is exposed.
+Datasets are represented by `dcat:Dataset` objects with the common mandatory fields and Dataset specific fields.
 
-## Representation Block
+As Datasets will be discovered by browsing an index, they need additional descriptive metadata for discovery. The following fields are mandatory:
 
-This section describes the format of any data received by a [data consumer](glossary.md#term-Data-Consumer) from this data set. The IB1 Trust FrameworkOpen Energy does 
-not mandate particular formats, so this section is guidance rather than specification.
+[dcterms:description](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/terms/description/)
+: Longer form description of this data set. This is used in combination with the title and tags when people search for data sets, so aim to include probable search words in the description.
 
-The only required element in this section is a key `mime` which should contain the 
-[media type](https://en.wikipedia.org/wiki/Media_type) of the returned data. At a bare minimum this allows a client to 
-load data into some kind of tooling. Depending on this value, other objects may be present.
+[dcat:version](https://www.w3.org/TR/vocab-dcat-3/#Property:resource_version)
+: Version number of the data set, this should preferably follow [semantic versioning](https://semver.org/) if possible. Versioning of the data set should be used to indicate changes in delivery mechanism, or in representation, rather than for changes in the underlying data. For example, this should not be used to differentiate between data sets from different years, rather it should be used to indicate whether a potential data consumer might need to alter how it processes any returned data. 
 
-### text/csv
+[dcat:versionNotes](https://www.w3.org/TR/vocab-dcat-3/#Property:resource_version_notes)
+: Notes used to explain any changes to this version.
 
-This type indicates that data is presented in CSV format. In this case, an optional key `csvw` may be defined, and 
-should contain valid [JSON-LD](glossary.md#term-JavaScript-Object-Notation-for-Linked-Data) following the [CSV for the Web](https://www.w3.org/TR/tabular-data-primer/) guidelines:
+[dcat:distribution](https://www.w3.org/TR/vocab-dcat-3/#Property:dataset_distribution)
+: URL of a `dcat:Distribution` for a downloadable file, see below for mandatory fields. Multiple Distribtions may be defined for the same data in different formats, but Scheme-conforming datasets will require a specific format.
```

Comment:
I don't think it's a given the scheme will require a single specific format - the scheme may define a set of acceptable formats for example